hls4ml/backends/vivado/passes/pointwise.py

```diff
@@ -75,8 +75,9 @@ def match(self, node):
 
     def transform(self, model, node):
         dim = node.__class__.__name__[-2:]  # '1D' or '2D'
-        pw_node = model.make_node('PointwiseConv' + dim, node.name, copy(node.attributes), node.inputs.copy())
-        pw_node.weights['bias'].data = node.weights['bias'].data
+        pw_node = model.make_node(
+            'PointwiseConv' + dim, node.name, copy(node.attributes), node.inputs.copy(), [x for x in node.outputs]
+        )
```

Comment:
What is the reason for the outputs? I do agree, though, that the weights don't need copying.